add HW5

Uaplexer opened the pull request:
I know that logging is not perfect

Uaplexer committed:
add HW5

Uaplexer committed:
Delete HomeWork3 directory

Uaplexer committed:
fix doc-string

Uaplexer committed:
Merge remote-tracking branch 'origin/feature/Work5' into feature/Work5

Uaplexer committed:
improve data insertion using named parameters and dict, add globals.py and env file management

Uaplexer committed:
improve validation and overall code

Uaplexer committed:
correct initial_db_setup_001.py

Uaplexer committed:
update with given instructions

vloooo commented on .idea/workspace.xml:
```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
should be added to gitignore

vloooo commented on HomeWork3/main.py:
```diff
@@ -1,187 +0,0 @@
-import logging
```
why is it deleted?

Uaplexer replied:
deleted by accident

Uaplexer replied:
because not used in main

Uaplexer committed:
add .idea to gitignore

Uaplexer committed:
last big update, improved imports, typing, more constants,
 better-organised transactions, changed globals.py to consts.py

 removed:
 - delete table func

Uaplexer committed:
Create README.md
